src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -619,6 +619,9 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
         .ignored {
           border: 1px solid var(--light-theme-disabled-color);
         }
+        .ignored img {
+          filter: grayscale(1);
+        }
```

Comment:
You should rebase, as this is from your other PR

Reply:
Sorry I tried to do that but must have done it wrong :( 

I've tested your suggestions locally and they work perfectly. Is it best to accept via the 'commit suggestion' link or try to start fresh and re-upload?

![image](https://user-images.githubusercontent.com/54028700/81114317-b57c5480-8f19-11ea-86b9-69c6c96448cc.png)

Reply:
It is easier to start fresh :-)